### ISSUE TITLE
Linkding Safari iOS Shortcut crashes with this URL

### ISSUE THREAD

**Issue by turkishmaid** [sissbruecker/linkding#346]
https://www.sciencedirect.com/science/article/pii/S0019357716300866

Environment: current Safari on recent iOS
Bookmarklet created as „Sharepad“ Shortcut as described in the docs. More precisely the linkding tab opens for a blink and is right away closed.

Looks like this is because of the strange „p“-character that is just about everywhere in this page. 

Too bad for a math addict :)

**Comment by sissbruecker** [comment]
Hmm, it's working fine for me. The title is not detected correctly but that's a different issue.

> Looks like this is because of the strange „p“-character that is just about everywhere in this page.

The content of the page should not matter for the shortcut or the page that it loads in Safari, the title and description are scraped on the server, not on your device.

It sounds more like the shortcut is not configured correctly. Did the shortcut work before? Can you bookmark other websites using the shortcut?

**Comment by turkishmaid** [comment]
Sure, I was able to use the shortcut to linkding several dozens of other pages, but not for exactly this single one. So I investigated a bit further. The issue was on an iPad with 120G of 128G full, which causes unwanted page reloads all the time. I tried the same on a new iPhone with plenty of memory and, oops, it worked. That page is complex, so it can well be yet another unwanted page reload, killing the dependent tab. Apologies for the false alert, Sascha. 

Nb.: I love linkding a lot 😍, was just up to write my own self-hosted bookmark service, but then found yours. Thanks a lot for giving it to us well maintained and even for free. There is, however, one little thing: I use have quite some content in descriptions and because iOS won't display the resize handle of the description field, it is quite a Sehschlitz to the text. I will open another issue for this if you don't mind. Keep up the good work, Sir 👍
